zql/Makefile

```diff
@@ -1,6 +1,4 @@
-GOIMPORTS = goimports
-PEGJS = pegjs
-PIGEON = pigeon
+deps = $(CURDIR)/deps
```

Comment:
CURDIR isn't great -- if I run `make -C zql` from the zq directory this puts the deps in a different place.

Reply:
@aswan: I see this:
```sh
$ pwd
/Users/noah/zq
$ make -C zql | head -1
mkdir -p /Users/noah/zq/zql/deps
```

Do you get different behavior?

Reply:
oops you're right, `-C` is fine, its `make -f zql/Makefile` that doesn't work with CURDIR.  But the latter doesn't work for lots of other reasons anyway and I don't think that's important...